bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -585,7 +581,7 @@ ViewChangeMsg* ViewsManager::exitFromCurrentView(SeqNum currentLastStable,
 bool ViewsManager::tryToEnterView(ViewNum v,
                                   SeqNum currentLastStable,
                                   SeqNum currentLastExecuted,
-                                  std::vector<PrePrepareMsg*>* outPrePrepareMsgsOfView) {
+                                  std::vector<std::shared_ptr<PrePrepareMsg>>* outPrePrepareMsgsOfView) {
```

Comment:
The original writer didnt do it right.
No reason in the world here ot pass ptr to vetor shared ptr.
I sugggest to pass the vector by reference.

Reply:
done